### PR TITLE
Two bug fixes + godep update

### PR DESCRIPTION
**Pull request by anthonybishopric**
This PR includes three commits, two of which represent bug fixes.

* When handling responses from Consul, check that `err != nil` prior to using `meta`. 
* Don't write (and actively remove) empty servicebuilder files.

The third commit simply updates the project's dependencies.

### CODE REVIEW

File: pkg/health/health.go [hunk 1]
```diff
@@ -40,10 +47,9 @@ func (h ConsulHealthChecker) WatchNodeService(nodename string, serviceID string,
 		case <-quitCh:
 			return
 		case <-time.After(1 * time.Second):
-			checks, meta, err := h.client.Health().Node(nodename, &api.QueryOptions{
-				WaitIndex: curIndex,
-				WaitTime:  h.WaitTime,
-			})
+			var err error
+			var checks []*api.HealthCheck
+			checks, curIndex, err = h.fetchNodeHealth(nodename, curIndex)
```
Comment (bwester) [inline pkg/health/health.go]: `curIndex` doesn't appear to be used anywhere... why produce it?

**Comment by anthonybishopric** [comment]
@petertseng correct, we were just holding it wrong.

File: pkg/health/health.go (hunk 1 above)
Comment (anthonybishopric) [inline pkg/health/health.go]: it's used on this line and passed to the health call, just as before.

Comment (bwester) [inline pkg/health/health.go]: Ah, of course! I was just misreading the code.

**Comment by bwester** [comment]
Looks fine to me
:ship:

**Pull request merged**